### ISSUE TITLE
Crash when connecting to CapitalOne

### ISSUE THREAD

**Issue by ekimia** [plaid/link#189]
*Please do not include your API `secret` or `public_key` in any issue.*
---
When submitting an issue for Link on the web or within a Webview, please include the following information:

|       Name       | Value |
|------------------|-------|
| Link `env`       |     development  |
| Browser          |       |
| Operating system |       iOS |

### A detailed description of the steps to reproduce the issue

* Install linkkit v1.0.7 
* login with CapitalOne
* verify your identity
* crash:

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[__NSCFString stringByReplacingOccurrencesOfString:withString:options:range:]: nil argument'
```

stack trace:
![image](https://user-images.githubusercontent.com/357513/28693384-2a0b339e-72d9-11e7-8ba7-a8f1dbeefa04.png)

### What you expected to see?
a result screen
### What you saw instead?
crash
### A screenshot that shows the state of the UI (if applicable)

---

When submitting an issue for [**Plaid Link iOS**](https://github.com/plaid/link/tree/master/ios)
please include the following information:

| Name              | Value              |
| ----------------- | ------------------ |
| Link `apiVersion` |                    |
| Link `env`        |                    |
| LinkKit version   |               1.0.7     |
| iOS version       |               10     |
| iOS device        |                    |

### A detailed description of the steps to reproduce the issue

* 
* 
*

**Comment by afh** [comment]
Hello @ekimia thank you for reporting this issue.

While we work on providing a fix for this issue you can work around the issue by configuring a clientName for LinkKit:

* Either by adding `clientName` to the `PLKPlaidLinkConfiguration` section in your `Info.plist` (see [here](https://github.com/plaid/link/tree/master/ios#required-plkplaidlinkconfiguration-items))
* or by setting the `clientName` property on your `PLKConfiguration` instance (see [here](https://github.com/plaid/link/blob/master/ios/LinkDemo-Swift/LinkDemo-Swift/ViewController.swift#L94))

**Comment by afh** [comment]
Hi @ekimia
once #196 is merged and LinkKit 1.0.8 released the issue you encountered is fixed.
